flink-connector-oracle-cdc/src/main/java/com/ververica/cdc/connectors/oracle/table/OracleTableSource.java

```diff
@@ -113,8 +113,7 @@ public ScanRuntimeProvider getScanRuntimeProvider(ScanContext scanContext) {
         RowType physicalDataType =
                 (RowType) physicalSchema.toPhysicalRowDataType().getLogicalType();
         MetadataConverter[] metadataConverters = getMetadataConverters();
-        TypeInformation<RowData> typeInfo =
-                scanContext.createTypeInformation(physicalSchema.toRowDataType());
```

Comment:
Nice catch, could you add a test for this? @ashulin

Reply:
I think about how to complete the unit test, because the object is not public.

Reply:
And I think Pg, Mongo also exist this issue ~_~

Reply:
I'll open a new PR to solve this problem of all table sources.